collections/_explainers/europska-zelena-dohoda.md

```diff
@@ -5,8 +5,8 @@ slug:        "europska-zelena-dohoda"
 published:   2023-06-20
 authors:
   - id: "katerina-kolouchova"
-  - ids: ["jirka-lnenicka", "kristi-psorn-zakopcanova"]
-    minor-role: "editácia"
+  - ids: "Viera Minarovičová", "Lenka Sedlačková"
```

Comment:
totok nefunguje a neviem, ako to urobiť tak, aby to bolo ok

Reply:
Zkus to ještě ohraničit hranatými závorkami (viz výše) a přidat pod minor-role: "preklad".

Reply:
A tedy Jirku a Kristi tam určitě nech u editace taky.